installation-and-setup/quickstart-windows.md

```diff
@@ -16,7 +16,8 @@ This tutorial describes how to install and start using Fiddler Everywhere on Win
 
 ## Prerequisites
 
-- Install  Windows 8 x64, Windows 10 x64. or Windows 11 x64.
+- Install Windows 10 x64 or Windows 11 x64.
+  >tip Windows 7 and 8.1 operating systems lack support for TLS 1.3, instead relying solely on the outdated TLS 1.2 protocol, which was declared insecure and reached its end-of-life status in 2023. To enhance security and align with modern practices, Fiddler Everywhere version 5.0.0 discontinues support for these outdated Windows versions. This decision ensures that Fiddler employs only secure TLS versions for in-app operations, such as all interactions with Fiddler cloud servers and account authentication. It's important to note that this change doesn't impact the in-app options for capturing sessions using various TLS versions.
```

Comment:
OS support is not equal to app support - e.g. Chrome works with TLS 1.3 on Windows 10 even though Windows 10 does not support officially TLS 1.3. I think it is better to say that Microsoft dropped support for the OS and all major browsers (Chromium and Firefox) dropped support too. In order to bring new features and improve security in FE, we need to drop support as well or something similar.